tests/smoke/conjugate_gradient.cpp

```diff
@@ -136,6 +136,39 @@ void ioProgram( const struct input &data_in, bool &success ) {
 	success = true;
 }
 
+using Ring = Semiring<
+	grb::operators::add< ScalarType >, grb::operators::mul< ScalarType >,
+	grb::identities::zero, grb::identities::one
+>;
+
+template<
+	Backend backend
+> RC computeNorm(
+	const grb::Vector< ScalarType, backend > &v,
+	grb::Vector< ScalarType, backend > &temp,
+	BaseScalarType &norm
```

Comment:
The code standard has that the output should be the first argument, not the last

Reply:
modification reverted